src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionIntTest.java

```diff
@@ -0,0 +1,244 @@
+package uk.gov.hmcts.darts.transcriptions.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.darts.authorisation.component.UserIdentity;
+import uk.gov.hmcts.darts.common.entity.CourtroomEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.TranscriptionEntity;
+import uk.gov.hmcts.darts.common.entity.TranscriptionStatusEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.repository.TranscriptionStatusRepository;
+import uk.gov.hmcts.darts.common.repository.UserAccountRepository;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.testutils.data.UserAccountTestData;
+import uk.gov.hmcts.darts.testutils.stubs.SuperAdminUserStub;
+import uk.gov.hmcts.darts.testutils.stubs.TranscriptionStub;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum;
+import uk.gov.hmcts.darts.transcriptions.model.GetTranscriptionDetailResponse;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+class TranscriptionIntTest extends IntegrationBase {
+
+    private static final String ENDPOINT_URL = "/admin/transcriptions?user_id=${USERID}&requested_at_from=${REQUESTED_FROM}";
+
+    private static final String ENDPOINT_URL_NO_DATE = "/admin/transcriptions?user_id=${USERID}";
+
+    @Autowired
+    private SuperAdminUserStub superAdminUserStub;
+
+    @Autowired
+    private TranscriptionStub transcriptionStub;
+
+    @Autowired
+    private UserAccountRepository userAccountRepository;
+
+    @Autowired
+    private TranscriptionStatusRepository transcriptionStatusRepository;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private UserIdentity userIdentity;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    void getTransactionsForUserWithoutDate() throws Exception {
```

Comment:
can we add a test for the 400 error when the endpoint is called without user_id param?